cmd/kube-controller-manager/app/options/options.go

```diff
@@ -272,6 +274,7 @@ func (s *KubeControllerManagerOptions) Flags(allControllers []string, disabledBy
 	fs.StringVar(&dummy, "insecure-experimental-approve-all-kubelet-csrs-for-group", "", "This flag does nothing.")
 	fs.StringVar(&s.OpenShiftContext.OpenShiftConfig, "openshift-config", s.OpenShiftContext.OpenShiftConfig, "indicates that this process should be compatible with openshift start master")
 	fs.MarkHidden("openshift-config")
+	fs.BoolVar(&s.OpenShiftContext.KubeAPIOverLocalhost, "unsupported-kube-api-over-localhost", false, "when set forces KCM to talk to the Kube API over localhost")
```

Comment:
when set makes KCM prefer talking to localhost kube-apiserver (when available) instead of LB